### PR TITLE
Converted three util classes to Kotlin

### PR DESCRIPTION
**Pull request by sriramr98**
First steps for issues #2357 

Changes: Converted three Util classes to kotlin

### CODE REVIEW

File: android/app/src/main/java/org/fossasia/openevent/core/auth/AuthUtil.kt [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+    const val VALID = 0
+    const val EMPTY = 1
+    const val INVALID = 2
+
+    private var authenticator: Authenticator? = null
+
+    @JvmStatic
+    val authorization: String
+        get() = formatToken(token!!)
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/core/auth/AuthUtil.kt]: SYIF(TM)

File: android/app/src/main/java/org/fossasia/openevent/core/auth/AuthUtil.kt [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        context.startActivity(intent)
+        ActivityCompat.finishAffinity(context as Activity)
+    }
+
+    private fun showProgressBar(progressBar: ProgressBar, show: Boolean) {
+        if (show) {
+            progressBar.visibility = View.VISIBLE
+        } else {
+            progressBar.visibility = View.GONE
+        }
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/core/auth/AuthUtil.kt]: Use ternary expression inside

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

File: android/app/src/main/java/org/fossasia/openevent/core/auth/AuthUtil.kt [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    const val INVALID = 2
+
+    private var authenticator: Authenticator? = null
+
+    @JvmStatic
+    val authorization: String
+        get() = formatToken(token)
+
+    private val token: String?
+        get() = SharedPreferencesUtil.getString(ConstantStrings.TOKEN, null)
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/core/auth/AuthUtil.kt]: If a token is nullable, authorization may as well

Comment (sriramr98) [reply android/app/src/main/java/org/fossasia/openevent/core/auth/AuthUtil.kt]: So you want the token not null ?

Comment (iamareebjamal) [reply android/app/src/main/java/org/fossasia/openevent/core/auth/AuthUtil.kt]: No, authorization should be nullable as well

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**